copy/tools/chartist.yml

```diff
@@ -0,0 +1,54 @@
+title: Chartist
+description: Simple responsive SVG charts
+logo: chartist.svg
+developer: null
+based_on: null
+licenses:
+  - type: open-source
+    title: MIT
+    link: 'https://github.com/chartist-js/chartist/blob/main/LICENSE-MIT'
+types:
+  - charts
+renders:
+  - svg
+features:
+  easy-to-customize: 0
```

Comment:
it's easy to customize 🙂

Reply:
I changed it to 1 which places it between other tools with similar number of GH stars, hope that's alright.